packages/workflow/src/config/routes.ts

```diff
@@ -111,6 +112,15 @@ export const getRoutes = () => {
         tags: ['api'],
         description: 'Unassign record endpoint'
       }
+    },
+    {
+      method: 'POST',
+      path: '/record/{id}/not-duplicate',
```

Comment:
```suggestion
      path: '/records/{id}/not-duplicate',
```
to keep the routes consistent